chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/pdf/DirectReviewSummaryStatisticsSectionPdf.java

```diff
@@ -70,5 +74,42 @@ public Table generateTable(LocalDate recent, LocalDate previous, EmailStatistics
         return table;
     }
 
+    @SuppressWarnings("resource")
+    @Override
+    public Document addTableEndNote(Document document, EmailStatistics recentEmailStatistics, EmailStatistics previousEmailStatistics) {
+        if (displayEndNote(recentEmailStatistics, previousEmailStatistics)) {
+            Text first = new Text("Not Available (N/A) ").setFont(SummaryStatisticsPdfDefaults.getDefaultBoldFont());
```

Comment:
Is this space at the end of the line needed? Or does it drop out automatically?

Reply:
Since Jim wanted that first phrase to be bold, I had to put two Text objects together.  The space is required either at the end of this Text or at the beginning of the second Text object.